x-pack/legacy/plugins/maps/public/layers/styles/vector/components/color/color_map_select.js

```diff
@@ -27,6 +35,53 @@ export class ColorMapSelect extends Component {
     };
   }
 
+  _renderColorMapToggle() {
+    const options = [
+      {
+        value: COLOR_MAP_TYPE.ORDINAL,
+        text: i18n.translate('xpack.maps.styles.dynamicColorSelect.quantitativeLabel', {
+          defaultMessage: 'Quantitative',
+        }),
+      },
+      {
+        value: COLOR_MAP_TYPE.CATEGORICAL,
+        text: i18n.translate('xpack.maps.styles.dynamicColorSelect.qualitativeLavel', {
+          defaultMessage: 'Qualitative',
+        }),
+      },
+    ];
+
+    const selectedValue = this.props.styleProperty.isOrdinal()
+      ? COLOR_MAP_TYPE.ORDINAL
+      : COLOR_MAP_TYPE.CATEGORICAL;
+
+    return (
+      <EuiToolTip
+        position="top"
+        content={i18n.translate(
+          'xpack.maps.styles.dynamicColorSelect.qualitativeOrQuantitativeTooltip',
+          {
+            defaultMessage:
+              'Choose `quantitative` to map the value as a number to a color on a range. Choose `qualitative` to map the value as a category to a color from a palette',
```

Comment:
@gchaps I could probably use some help with some good help-text here 😃 Any ideas to make this read better?

Reply:
Instead of a tooltip, how about EuiFormRow help text for the selected value?

Reply:
Do either of these work?

Choose `Quantitative` to map the value to a number in a color range, or `Qualitative` to map to a category in a color palette.

Choose `Quantitative` to map by number in a color range, or `Qualitative` to categorize by color palette.

Reply:
thx @gchaps !

to avoid some clutter in the UX (which is getting really long), we were thinking if we could avoid the help-text if the select-box uses "As number" and "As category" for the options. It'd still using your help-text in the aria-label.

![image](https://user-images.githubusercontent.com/1833023/75358003-38c95c00-5880-11ea-9df4-65333b7c421a.png)

Reply:
That wording is much clearer than using quantitative and qualitative.

But should it be "By number" and "By category"?

Reply:
@gchaps maybe (?)  :)  not super familiar with the nuances here.

It's basically to express the difference in how the value should be interpreted. Should the value be interpreted as a number on a range, then we can map it to a continuous color-range. Should the value be interpreted as a category, then we just assign a color-value.

I can make this change to 'by', but 'as' seems to fit a little better by what it's trying to express.

Reply:
Got it.  'as' is the better choice.